panic: open pet: no such file or directory

## Problem statement

$GOPATH/src/github.com/go-swagger/go-swagger/examples/2.0/petstore/server/

go run petstore.go

```
panic: open pet: no such file or directory

goroutine 1 [running]:
github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/validate.NewSchemaValidator(0xc4201998c0, 0x886380, 0xc420071c00, 0xc4203a21d0, 0x3, 0xb104e0, 0xc4202b2de0, 0xc42066d6b0)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/validate/schema.go:55 +0xf9b
github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware.newUntypedParamBinder(0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware/parameter.go:46 +0x1dd
github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware.newUntypedRequestBinder(0xc4202574a0, 0xc420071c00, 0xb104e0, 0xc4202b2de0, 0xc4202575f0)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware/request.go:39 +0x1ac
github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware.(*defaultRouteBuilder).AddRoute(0xc420256bd0, 0x8ecc37, 0x4, 0xc420343197, 0x9, 0xc4200fcee0)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware/router.go:252 +0x699
github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware.DefaultRouter(0xc420075db0, 0xb10600, 0xc420021380, 0x8832c0, 0xc420362d10)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware/router.go:136 +0x36e
github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware.NewRouter(0xc420020800, 0xb09e40, 0xc420362d10, 0xc420362d10, 0x0)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware/router.go:68 +0xfd
github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware.(*Context).RoutesHandler(0xc420020800, 0x90d500, 0x0, 0x0)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware/context.go:554 +0x7d
github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware.(*Context).APIHandler(0xc420020800, 0x90d500, 0x0, 0x0)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware/context.go:545 +0xec
github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware.ServeWithBuilder(0xc420075db0, 0xc4201ef8f0, 0x90d500, 0xc420362736, 0x6)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware/context.go:218 +0x5f
github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware.Serve(0xc420075db0, 0xc4201ef8f0, 0x3, 0x8f1ccf)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/vendor/github.com/go-openapi/runtime/middleware/context.go:211 +0x41
github.com/go-swagger/go-swagger/examples/2.0/petstore/server/api.NewPetstore(0x0, 0x875260, 0x1, 0x0)
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/examples/2.0/petstore/server/api/petstore.go:45 +0x21f
main.main()
	/home/wade.lee/goWorkProject/src/github.com/go-swagger/go-swagger/examples/2.0/petstore/server/petstore.go:25 +0x26
exit status 2
```

## Swagger specification
is  swagger.json ?
## Environment
centos 7
x86_64 x86_64 x86_64 GNU/Linux
go version go1.8 linux/amd64

--- a/examples/2.0/petstore/server/api/petstore.go
+++ b/examples/2.0/petstore/server/api/petstore.go
@@ -324,7 +324,7 @@ var swaggerJSON = `{
     "petInput": {
       "allOf": [
         {
-          "$ref": "pet"
+          "$ref": "#/definitions/pet"
         },
         {
           "required": [

